grails-app/services/com/asinha/domain/PaymentService.groovy

```diff
@@ -25,4 +32,13 @@ class PaymentService {
         payment.save(failOnError: true)
         return payment
     }
+
+    public Payment confirmPayment(paymentId) {
+        Payment payment = Payment.get(paymentId)
+        payment.status = PaymentStatus.PAID
+        payment.paymentDate = new Date()
+        payment.lastUpdate = new Date()
```

Comment:
Precisa forçar essa atualização manualmente?

Reply:
descobrimos depois que o lastUpdated é automatico e corrigimos pra frente, mas vou corrigir aqui já